scripts/deploy-on-cozy

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+set -euo pipefail
+
+JOB_UPDATE_INT="87a668f7-eff1-422d-aa84-92b3bcc62c8f"
+JOB_UPDATE_STAGING="ad27f2f6-63d9-4a16-ab62-e25c957875b5"
+JOB_UPDATE_PROD="87a668f7-eff1-422d-aa84-92b3bcc62c8f"
```

Comment:
I would keep only the `INT` concern since we won't use the others here via the CI.